Gemfile

```diff
@@ -20,7 +20,7 @@ group :default do
   gem 'hashie'
 
   # Communications with JSON APIs, allows us to begin migration to the new Sequencescape API
-  gem 'json_api_client', github: 'sanger/json_api_client', branch: 'merge_upstream'
+  gem 'json_api_client', github: 'sanger/json_api_client', branch: 'v1.21.0a'
```

Comment:
See https://github.com/sanger/json_api_client/pull/8 for the changes pulled in. Its mostly just improved error handling, and a couple of create/update methods. Importantly it also relaxes the Faraday constraint,  although doesn't move us on to 2.0.

I do wish there were better options available.